api/auth/approle/approle.go

```diff
@@ -125,7 +125,7 @@ func (a *AppRoleAuth) Login(ctx context.Context, client *api.Client) (*api.Secre
 
```

Comment:
I think you may have missed the `gcp` submodule

Reply:
Thanks, good catch 👍